core/src/main/resources/error/error-classes.json

```diff
@@ -35,6 +35,12 @@
     ],
     "sqlState" : "22003"
   },
+  "BINARY_ARITHMETIC_CAUSE_OVERFLOW" : {
```

Comment:
Let's make it shorter `BINARY_ARITHMETIC_OVERFLOW` similarly to other error class names in the file.